backoffice/utils_proctorU_api.py

```diff
@@ -59,8 +59,20 @@ def get_mongo_reports(course_key):
     cached_api_proctoru = get_proctoru_api_collection()
 
     mongo_datas = cached_api_proctoru.find_one()
+    if not mongo_datas:
+        pb = {"id": "Mongo database is empty, come back later",
+                         "start": None,
+                         "end": None}
+        return {"warn": pb}, datetime.datetime.now()
```

Comment:
Je trouve cette valeur de retour très étrange : 

1. Retourner un dictionnaire ne contenant qu'une seule clé porte à confusion ; les rapports aggrégés ne peuvent-ils pas avoir une clé appelée "warn" ?
2. Qu'est-ce qui justifie que id=="Mongo database is empty, come back later"
3. Pourquoi la date de dernière mise à jour est-elle égale à `now` alors que la base de donnée n'a jamais été mise à jour ?

J'ai l'impression qu'il vaudrait mieux lever une exception personnalisée, du genre ProctoruCacheUnavailable.